packages/client-api-schema/src/types.ts

```diff
@@ -160,9 +160,18 @@ export interface SiteBudget {
   inUse: Balance;
   direct: Balance;
 }
+
+export interface BudgetRequest extends Balance {
+  site: string;
```

Comment:
Rename to `domain`?

Reply:
currently keeping as `site` to keep it consistent with all other `budget` related entities. `domain` probably is a better name tho